src/views/List.jsx

```diff
@@ -24,6 +24,9 @@ export function List({ data, listToken }) {
 
 	return (
 		<>
+			{/* The height of the green section of the navigation tab is: h-20 (5rem);
+		The button is sticking out by -mt-6 (1.5rem);
+		You'll need 6.5rem of empty space at the bottom of your page so that the user can see their entire list.  */}
```

Comment:
Is this a dead comment? Or a comment for a different dev?

Reply:
@spittel The deadcode on this PR is from code outside of the scope of this PR [the code you saw were outdated]. But you're right, we do have a bunch of commented out CSS that we need to deleted. I'm working on that and it should be gone when we merge.